sktime/datatypes/_adapter/polars.py

```diff
@@ -101,7 +123,9 @@ def get_index_names(cols):
     return obj
 
 
-def check_polars_frame(obj, return_metadata=False, var_name="obj", lazy=False):
+def check_polars_frame(
```

Comment:
the changes to this function change - and break - its current use for the `Table` scitype.

Reply:
What are the possible things that could break ??, I'll have a look at them